handlers/admin/data-contributions.js

```diff
@@ -1,13 +1,16 @@
 'use strict';
 
 const Joi = require('joi');
+const DataCategory = require('../../models/data-category');
 const DataContribution = require('../../models/data-contribution');
 
 
 function editDataContribution(request, reply) {
   const data = {
     approved: request.payload.approved,
-    curation_comments: request.payload.curation_comments,
+    trial_id: request.payload.trial_id || null,
+    data_category_id: request.payload.data_category_id,
+    curation_comments: request.payload.curation_comments || null,
```

Comment:
If by check you mean "remove the `|| null` and see what breaks", taking a couple minutes, then yes. This type of weird-ish code has a tendency of living on in my experience, as people don't want to change what they don't understand :smile: